Code/Engine/ShaderCompilerHLSL/ShaderCompilerHLSL.cpp

```diff
@@ -346,6 +346,39 @@ ezSharedPtr<ezShaderConstantBufferLayout> ezShaderCompilerHLSL::ReflectConstantB
   return pLayout;
 }
 
+ezResult ezShaderCompilerHLSL::AddFakeSetAssignments(ezShaderProgramData& inout_Data, ezGALShaderStage::Enum Stage, ezLogInterface* pLog)
```

Comment:
Should be called bind group and not set

Reply:
Oh right, 'set' really is an inconvenient word to search for. Fixed!